tools/cargo_makepad/src/android/compile.rs

```diff
@@ -56,6 +56,7 @@ fn manifest_xml(label:&str, class_name:&str, url:&str)->String{
         <uses-permission android:name="android.permission.BLUETOOTH_CONNECT"/>
         <uses-permission android:name="android.permission.CAMERA"/>
         <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
+        <uses-permission android:name="android.permission.USE_BIOMETRIC" />
```

Comment:
this is only here since we don't yet have a standardized way for a third-party app to add permissions